src/plots/cartesian/graph_interact.js

```diff
@@ -2099,8 +2099,7 @@ fx.dragElement = function(options) {
         if(options.doneFn) options.doneFn(gd._dragged, numClicks);
 
         if(!gd._dragged) {
-            var e2 = document.createEvent('MouseEvents');
-            e2.initEvent('click', true, true);
+            var e2 = new MouseEvent('click', { 'button': e.button });
```

Comment:
The *new style* is supported in IE9, according to https://developer.mozilla.org/en/docs/Web/API/MouseEvent

if that's the case, then :+1: for the new style.